kyuubi-hive-beeline/src/main/resources/BeeLine.properties

```diff
@@ -154,73 +154,76 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
+\                                    defaults to 1000; only applicable if --incremental=true\n \
+\                                    and --outputformat=table.\n\n \
+\  --truncateTable=[true|false]      Truncate table column when it exceeds length.\n \
+\  --delimiterForDSV=DELIMITER       Specify the delimiter for delimiter-separated values output format (default: |).\n \
+\  --isolation=LEVEL                 Set the transaction isolation level.\n \
+\  --nullemptystring=[true|false]    Set to true to get historic behavior of printing null as empty string.\n \
+\  --maxHistoryRows=MAXHISTORYROWS   The maximum number of rows to store beeline history.\n \
+\  --delimiter=DELIMITER             Set the query delimiter; multi-char delimiters are allowed, but quotation\n \
+\                                    marks, slashes, and -- are not allowed; defaults to ;\n\n \
+\  --convertBinaryArrayToString=[true|false]\n \
+\                                    Display binary column data as string or as byte array.\n\n \
+\  --python-mode                     Execute python code/script.\n \
+\  --help                            Display this message.\n \
 \n \
 \  Example:\n \
-\   1. Connect using simple authentication to HiveServer2 on localhost:10000\n \
-\   $ beeline -u jdbc:hive2://localhost:10000 username password\n\n \
-\   2. Connect using simple authentication to HiveServer2 on hs.local:10000 using -n for username and -p for password\n \
-\   $ beeline -n username -p password -u jdbc:hive2://hs2.local:10012\n\n \
-\   3. Connect using Kerberos authentication with hive/localhost@mydomain.com as HiveServer2 principal\n \
-\   $ beeline -u "jdbc:hive2://hs2.local:10013/default;principal=hive/localhost@mydomain.com"\n\n \
-\   4. Connect using SSL connection to HiveServer2 on localhost at 10000\n \
-\   $ beeline "jdbc:hive2://localhost:10000/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"\n\n \
+\   1. Connect using simple authentication to KyuubiServer on localhost:10009\n \
+\   $ kyuubi-beeline -u jdbc:kyuubi://localhost:10009 username password\n\n \
+\   2. Connect using simple authentication to KyuubiServer on kyuubi.local:10009 using -n for username and -p for password\n \
+\   $ kyuubi-beeline -n username -p password -u jdbc:kyuubi://kyuubi.local:10009\n\n \
+\   3. Connect using Kerberos authentication with hive/localhost@mydomain.com as KyuubiServer principal\n \
+\   $ kyuubi-beeline -u "jdbc:kyuubi://kyuubi.local:10009/default;principal=hive/localhost@mydomain.com"\n\n \
+\   4. Connect using SSL connection to KyuubiServer on localhost at 10009\n \
+\   $ kyuubi-beeline "jdbc:kyuubi://localhost:10009/default;ssl=true;sslTrustStore=/usr/local/truststore;trustStorePassword=mytruststorepassword"\n\n \
 \   5. Connect using LDAP authentication\n \
-\   $ beeline -u jdbc:hive2://hs2.local:10013/default <ldap-username> <ldap-password>\n \
```

Comment:
I remember this example does not work (I haven't verified it by myself)
https://issues.apache.org/jira/browse/HIVE-21239

Reply:
Yes, I just checked this JIRA as well, and according to the reply within it, when using LDAP authentication for the connection, `-n <ldap-username> -p <ldap-password>` is required for proper functioning. Furthermore, I examined the master branch code of Hive: `BeeLine.properties`, and indeed, it has already been modified to `$ beeline -n ldap-username -p ldap-password -u "jdbc:hive2://localhost:10000/default"`.